engine/src/main/java/net/jqwik/engine/properties/configurators/WhitespaceConfigurator.java

```diff
@@ -0,0 +1,25 @@
+package net.jqwik.engine.properties.configurators;
+
+import net.jqwik.api.arbitraries.CharacterArbitrary;
+import net.jqwik.api.arbitraries.StringArbitrary;
+import net.jqwik.api.configurators.ArbitraryConfiguratorBase;
+import net.jqwik.api.constraints.Whitespace;
+import net.jqwik.engine.properties.arbitraries.DefaultStringArbitrary;
+
+public class WhitespaceConfigurator extends ArbitraryConfiguratorBase {
+
+	public StringArbitrary configure(StringArbitrary arbitrary, Whitespace whitespace) {
+		return arbitrary.withChars(DefaultStringArbitrary.WHITESPACE_CHARS);
+	}
+
+	public CharacterArbitrary configure(CharacterArbitrary arbitrary, Whitespace whitespace) {
+		CharacterArbitrary result = arbitrary;
+
+		for (char c : DefaultStringArbitrary.WHITESPACE_CHARS) {
+			result = result.between(c, c);
```

Comment:
Problem is that `DefaultCharacterArbitrary.between()` does not add the range but replaces a previous range :-( To accomplish what you want will probably require a change to `DefaultCharacterArbitrary`.

Reply:
`DefaultCharacterArbitrary` doesn't look too complicated, but I see several design-decisions coming-up that I don't want to do with comment-ping-pong.  
Could you describe in advance the design goals you have in mind?  
Or should we simply go without this enhancement and instead make the documentation more precise (e.g. point out that only one (the last?) of multiple of the constraint annotations will be respected when generating chars)?

Reply:
I updated DefaultCharacterArbitrary to add up chosen ranges and enhanced it with the capability to allow a set of chars. see `DefaultCharacterArbitrary.with(chars... allowedChars)`. This should suit the purpose I hope.